projects/packages/blaze/src/class-blaze.php

```diff
@@ -161,6 +206,47 @@ public static function should_initialize() {
 		return apply_filters( 'jetpack_blaze_enabled', $should_initialize );
 	}
 
+	/**
+	 * Get URL to create a Blaze campaign for a specific post.
+	 *
+	 * This can return 2 different types of URL:
+	 * - Calypso Links
+	 * - wp-admin Links if access to the wp-admin Blaze Dashboard is enabled.
+	 *
+	 * @param int $post_id Post ID.
+	 *
+	 * @return array An array with the link, and whether this is a Calypso or a wp-admin link.
```

Comment:
I'm not a huge fan of returning an array here just to get the info on whether we should link outside of wp-admin or not, I would rather return just the URL. That said, this seemed cleaner than having to check the string returned by this method later on just to find out if it was an external link or not.